core/src/main/java/ai/timefold/solver/core/api/solver/SolverJobBuilder.java

```diff
@@ -70,6 +70,17 @@ default SolverJobBuilder<Solution_, ProblemId_> withProblem(Solution_ problem) {
     SolverJobBuilder<Solution_, ProblemId_>
             withFinalBestSolutionConsumer(Consumer<? super Solution_> finalBestSolutionConsumer);
 
+    /**
+     * Sets the initialized solution consumer, which is called before starting the first
+     * {@link ai.timefold.solver.core.impl.localsearch.LocalSearchPhase} or
+     * {@link ai.timefold.solver.core.impl.phase.custom.CustomPhase} phases.
```

Comment:
This is not what was agreed, is it?
First local search, yes. First custom phase, no. Custom phase may still be initializing.

Also, arguably, the definition deserves a better explanation:

"Sets the consumer of the first initialized solution. First initialized solution is the solution at the end of the last phase that immediately precedes the first local search phase. This solution marks the beginning of actual optimization."

Also, I'd consider changing the name of the concept from "initialized solution" to "first initialized solution", to clearly explain what is the significance of it.

Reply:
I thought we had agreed on the first custom phase as well, which could be a custom local search.

![image](https://github.com/TimefoldAI/timefold-solver/assets/11961659/667d6ba8-86d6-4143-86f2-6015c26c6c93)

Reply:
I believe that the part that you're sharing here was only a by-product of the conversation.
At the bottom of the document, you see proposal C saying that we only use the first Local Search.

We can not use a custom phase, because custom phase _could_ be some sort of local search, but it could also be a custom construction heuristic. The only place where we can be _sure_ is before the first actual local search.

Reply:
Okay. It makes sense.